src/main/java/com/openmrs/migrator/core/services/impl/DataBaseServiceImpl.java

```diff
@@ -42,4 +55,18 @@ public void createDatabase(String databaseName) {
         String.format(
             "drop database if exists %s; create database %s;", databaseName, databaseName));
   }
+
+  @Override
+  public List<String> getDatabases(String password) throws IOException {
+    Process p =
+        Runtime.getRuntime()
+            .exec(new String[] {"mysql", "-u", "root", "-p" + password, "-e", "show databases"});
```

Comment:
We are using root here instead of the username from settings - best not to hard code this.


Lets also generalise this into one function to "run" SQL commands into one function. Seems we will need to run this for many situations - we also have the CommandService for this kind of thing. Please use or extend it

Reply:
done